sqlite-utils: add page (#8088)
tldr-pages/tldr: pages/common/sqlite-utils.md

# sqlite-utils

> Command-line tool used to manipulate SQLite databases in a number of different ways.
> More information: <https://sqlite-utils.datasette.io/en/stable/cli.html>.

- Create a database:

> `sqlite-utils create-database person.db`

Reviewer: ```suggestion
`sqlite-utils create-database {{path/to/database.db}}`
```

- Create a table:

> `sqlite-utils create-table person.db people id integer name text height float photo blob --pk id`

Reviewer: ```suggestion
`sqlite-utils create-table {{path/to/database.db}} {{table_name}} {{id integer name text height float photo blob --pk id}}`
```

- Upsert a record:

> `echo '[ {"id": 1, "name": "Linus Torvalds"}, {"id": 2, "name": "Steve Wozniak"}, {"id": 3, "name": "Tony Hoare"} ]' | sqlite-utils upsert person.db people - --pk id`

Reviewer: ```suggestion
`{{echo '[ {"id": 1, "name": "Linus Torvalds"}, {"id": 2, "name": "Steve Wozniak"}, {"id": 3, "name": "Tony Hoare"} ]'}} | sqlite-utils upsert {{path/to/database.db}} {{table_name}} - {{--pk id}}`
```

- Select records:

> `sqlite-utils rows person.db people`

Reviewer: ```suggestion
`sqlite-utils rows {{path/to/database.db}} {{table_name}}`
```

- Delete a record:

> `sqlite-utils query person.db "delete from people where name = 'Tony Hoare'"`

Reviewer: ```suggestion
`sqlite-utils query {{path/to/database.db}} {{"delete from people where name = 'Tony Hoare'"}}`
```

- Drop a table:

> `sqlite-utils drop-table person.db people`

Reviewer: ```suggestion
`sqlite-utils drop-table {{path/to/database.db}} {{table_name}}`
```

> `sqlite-utils query {{path/to/database.db}} {{"delete from {{table_name}} where name = 'Tony Hoare'"}}`

Reviewer 2: By convention we use quotes `"` outside the token syntax. Also, the token syntax can't be nested, it's a markdown spec
```suggestion
`sqlite-utils query {{path/to/database.db}} "{{delete from table_name where name = 'Tony Hoare'}}"`
```

Reviewer 2: I'll apply this quick fix, then merge.
